core/components/text-editor.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
```

Comment:
The idea of `core/components` give you the hint that is actually a resource. This can go to `resources/inputs` or similar folder